.cicd/pipeline.yml

```diff
@@ -84,4 +106,93 @@ steps:
       - "queue=mac-anka-large-node-fleet"
     timeout: $BUILD_TIMEOUT
     skip: $SKIP_MOJAVE
-    
\ No newline at end of file
+
+  - wait:
+    continue_on_failure: true
+
+  # - command: |
+  #     echo '+++ :compression: Extracting Test Metrics Code'
+  #     tar -zxf .cicd/metrics/test-metrics.tar.gz
+  #     echo '+++ :javascript: Running test-metrics.js'
+  #     node --max-old-space-size=32768 test-metrics.js
+  #   label: ":bar_chart: Test Metrics"
+  #   agents:
+  #     queue: "automation-eos-builder-fleet"
+  #   timeout: 10
+  #   soft_fail: true
+
+  # - wait
+
+  - label: ":centos: Centos 7 - PB"
```

Comment:
This does not follow the label guidelines established [here](https://github.com/EOSIO/eos-buildkite-pipelines/pull/27).

Reply:
This was not addressed.